user_guide/admin_capabilities.rst

```diff
@@ -0,0 +1,461 @@
+third-party packages installed  and is usually used by the system administrators and integrators only.
+
+
+.. _admin-capabilities-system-config:
+
+System configuration
+--------------------
+
+The capability enables users to access the :ref:`system configuration page <admin-configuration>`. If it is set to 
+*"System"*, the system administrator can localize the system, change the display and tracking settings, and make other 
+important changes.
+
+
+.. _admin-capabilities-tags:
+
+Tag assign/unassign
+-------------------
+
+The capability enables users to assign/unassign :ref:`tags <user-guide-tags>` - non-hierarchical keywords or phrases 
+assigned to records. They provide additional information about records and are visible to all the system users.
+You can assign/unassign tags from the grid or from a View page of the record.
+
+Tags can be successfully utilized by all the users.
+
+
+.. _admin-capabilities-tags-all:
+
+Unassign all tags from entities
+-------------------------------
+
+The capability enables users to unassign tags added to the records by other users. The capability is only 
+meaningful if *"Tag assign/unassign"* has been enabled (set to *"System"*). If *"Tag assign/unassign"* has been enabled, 
+and *"Unassign all tags from entities"* is set to *"None"*, the users will only be able to unassign tags added by 
+themselves -  this way you can restrict users from deleting tags of other users. The capability is usually available to 
+team leads, heads of departments and managers..
+
+.. _admin-capabilities-unshare-grid:
+
+Unshare grid view
+-----------------
+
+If this capability is set to *"System"*, users with the role can unshare grids previously 
+:ref:`shared <admin-capabilities-share-grid>` by themselves. This capability is usually available to all the users, who 
+work with grids.
+
+
+.. _admin-capabilities-view-sql:
+
+View SQL query of a report/segment
+----------------------------------
+
+If this capability is set to *"System"*, users with the role can see the SQL request sent to the system for a 
+report/segment. 
+
+Usually this capability is only granted to the system administrators, so they can check if the report has been developed 
+correctly. The *"Show SQL Query"* link will appear below the report.
+
+|
+
+.. image:: ./img/roles/sql_show.png
+
+|
+
+
+The setting will only work if the functionality has been enabled in the *"System Configuration --> Display Settings --> 
+Report settings*". 
+
+|
+
+.. image:: ./img/roles/sql_setting.png
+
+|
+
+
+.. _admin-capabilities-workflow:
+
+Workflow manipulations
+----------------------
+
+If this capability is set to *"System"*, users with the role can manage the records, involved in the 
+:ref:`workflows <user-guide-workflow-management-basics>`, otherwise the user may be able to see and edit the records, 
+but not to change their status within the workflow. This may be used to restrict the users from changing the record 
+status.
```

Comment:
Edit: If this capability is set to *"System,"* users with the role can manage the records associated with :ref:`workflows <user-guide-workflow-management-basics>`. Otherwise, users may be able to see and edit records, but will not be able to change their status within the workflow. This capability may be used to restrict users from changing the status of records.